LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016 OSS Friday
+Copyright (c) 2016 Open Source Friday
```

Comment:
I think that they started the project in 2016 though so arguable on when they want to do the year?